.tekton/network-observability-console-plugin-push.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     build.appstudio.openshift.io/repo: https://github.com/netobserv/network-observability-console-plugin?rev={{revision}}
     build.appstudio.redhat.com/commit_sha: '{{revision}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
-    build.appstudio.openshift.io/build-nudge-files: "hack/container_digest.sh"
+    build.appstudio.openshift.io/build-nudge-files: "hack/nudging/container_digest.sh"
```

Comment:
@OlivierCazade is it really needed to have this annotation here in console-plugin repo? Isn't it something for the operator repo only?

Reply:
This annotation is required for each build triggering a nudge PR.

By default, nudging PR will only look for yaml and Dockerfile.